.codesandbox/ci.json

```
@@ -8,10 +8,10 @@
     "packages/grid/x-grid-data-generator"
   ],
   "publishDirectory": {
-    "@mui/x-license-pro": "packages/x-license",
-    "@mui/x-data-grid": "packages/grid/data-grid",
-    "@mui/x-data-grid-pro": "packages/grid/x-grid",
-    "@mui/x-data-grid-generator": "packages/grid/x-grid-data-generator"
+    "@mui/x-license-pro": "packages/x-license/dist",
```

we name this folder `build` in the main repository. If it's easy to change, I would propose we keep the same conversion
```suggestion
    "@mui/x-license-pro": "packages/x-license/build",
```

---

Yeah, I noticed that. Now it's named `build`.